server.go

```diff
@@ -138,6 +138,15 @@ func (s *server) Serve() chan error {
 		}
 		mux.HandleFunc("/reframe", reframe)
 	}
+
+	delegated, err := NewDelegatedTranslator(s.doFind)
+	if err != nil {
+		ec <- err
+		close(ec)
+		return ec
+	}
+	mux.Handle("/v1", delegated)
```

Comment:
we changed path prefix to `/routing/v1`

```suggestion
	mux.Handle("/routing/v1", delegated)
```